IdentityCore/tests/MSIDAADWebviewFactoryTests.m

```diff
@@ -114,6 +115,7 @@ - (void)testResponseWithURL_whenURLSchemeNotMsauth_shouldReturnAADAuthResponse
     
     NSError *error = nil;
     __auto_type response = [factory responseWithURL:[NSURL URLWithString:@"redirecturi://somepayload?code=authcode&cloud_instance_host_name=somename"]
+                                        redirectUri:@"redirecturi"
```

Comment:
nit: can we actually use redirectUri in tests for correctness? This is a scheme.